src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return 'name: ' + this.name + ', chip version: '
+      + this.version + ', wheels: ' + this.wheels;
+    },
+
+    get location() {
+      return `${robot.name}: x=${robot.coords.x}, y=${robot.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value > 0) {
+        robot.coords.y += value;
+      }
+
+      return robot;
+    },
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
    },
```
try to operate with this

Reply:
everywhere